deploy/k8s/charts/trustification-infrastructure/Chart.yaml

```diff
@@ -20,7 +20,7 @@ home: https://trustification.io
 dependencies:
   - name: keycloak
     repository: https://charts.bitnami.com/bitnami
-    version: 18.2.1
+    version: 22.2.1
```

Comment:
I feel like this update is out of scope for this PR.
If confirmed, please remove it.

Reply:
yes I will change it.